TourMate/TourMate/Frontend/Plan/Models/Accommodation.swift

```diff
@@ -47,6 +47,21 @@ class Accommodation: Plan {
                    versionNumber: versionNumber)
     }
 
+    override func copy() -> Plan {
+        Accommodation(id: id,
+                 tripId: tripId,
+                 name: name,
+                 startDateTime: startDateTime,
+                 endDateTime: endDateTime,
+                 status: status,
+                 creationDate: creationDate,
+                 modificationDate: modificationDate,
+                 ownerUserId: ownerUserId,
+                 modifierUserId: modifierUserId,
+                 versionNumber: versionNumber,
+                 location: location)
+    }
```

Comment:
I needed to add copy because my list were getting passed by reference, then any modifications will affect the view itself